ratis-logservice/src/main/java/org/apache/ratis/logservice/tool/VerificationTool.java

```diff
@@ -176,6 +176,7 @@ public static void main(String[] args) throws IOException {
           }
           waitForCompletion(futures);
         }
+        client.close();
```

Comment:
Because client implements `AutoCloseable`, so it has a `close()` function should be called.

Reply:
we can use `try (LogServiceClient client = new LogServiceClient(tool.metaQuorum)) { ... }`

Reply:
@runzhiwang Yes. Pushed a commit to use try-with-resources pattern.